app/controllers/api/v1/featured_tags_controller.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+class Api::V1::FeaturedTagsController < Api::BaseController
+  before_action -> { doorkeeper_authorize! :read, :'read:featured_tags' }, only: [:index, :show]
+  before_action -> { doorkeeper_authorize! :write, :'write:featured_tags' }, except: [:index, :show]
+
+  before_action :require_user!
+  before_action :set_featured_tags, only: :index
+  before_action :set_featured_tag, except: [:index, :create]
+
+  def index
+    render json: @featured_tags, each_serializer: REST::FeaturedTagSerializer
+  end
+
+  def show
```

Comment:
I think `show` is probably not going to be used by apps.

Reply:
This is also true.